turbomoleio/input/tests/test_define.py

```diff
@@ -683,6 +697,16 @@ def test_set_basis_error(self, dr_data):
         with pytest.raises(DefineParameterError):
             dr_data.dr._set_basis("all", "wrong basis")
 
+    def test_set_basis_error_2(self, dr_data):
+        dr_data.expect_mock.side_effect = [0, 2]
+        with pytest.raises(DefineParameterError):
```

Comment:
Do you think it's worth to add a regex match to relevant part of the error message here ?

Reply:
I added them, but I think now this is the only place where it is done in this module. So should we consider adding this somewhere else as well?